static/app/views/dashboardsV2/detail.spec.jsx

```diff
@@ -19,6 +19,10 @@ import * as types from 'sentry/views/dashboardsV2/types';
 import ViewEditDashboard from 'sentry/views/dashboardsV2/view';
 import {OrganizationContext} from 'sentry/views/organizationContext';
 
+jest.mock('sentry/components/charts/worldMapChart', () => ({
+  WorldMapChart: () => null,
+}));
+
```

Comment:
This was complaining about setState after unmounting... none of the tests seemed to need the actual component so...